code/game/objects/items/ego_weapons/non_abnormality/rcorp.dm

```diff
@@ -143,12 +143,21 @@
 	can_charge = FALSE
 	weapon_weight = WEAPON_HEAVY // No dual wielding
 	pin = /obj/item/firing_pin/implant/mindshield
+	//None of these fucking guys can use Rcorp guns
+	var/list/banned_roles = list("Raven Squad Captain", "Reindeer Squad Captain","Rhino Squad Captain",
```

Comment:
Shouldn't this be an include list instead of an exclude list?

Reply:
Nah. I'm cool with agents using it.